app/cbt/utils/emacs.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import subprocess
+import typing
+
+# Concatenate strings using an Emacs subprocess as a show of
+# dominance. Only use with known static data to make 100% sure nobody
+# can somehow escape the quotes and run arbitrary code.
+def emacs_concat_string(str1: str, str2: str, sep: str=""):
```

Comment:
This doesn't appear to support sql injection or XSS. Please update to correct.

Reply:
A proof of concept arbitrary code execution can be achieved by running

`emacs_concat_string("hello\" (current-time-string)", "\"there")`

I hope that is sufficiently awful?

Reply:
911d6ae adds a meme that makes use of arbitrary code execution.